Mathlib/Data/NNRat/Defs.lean

```diff
@@ -45,13 +46,23 @@ un``@[simp]``ed):
 
 open Function
 
-deriving instance CanonicallyOrderedCommSemiring for NNRat
-deriving instance CanonicallyLinearOrderedAddCommMonoid for NNRat
+instance Rat.instZeroLEOneClass : ZeroLEOneClass ℚ where
+  zero_le_one := rfl
+
+instance Rat.instPosMulMono : PosMulMono ℚ where
+  elim := fun r p q h => by
+    simp only [mul_comm]
+    simpa [sub_mul, sub_nonneg] using Rat.mul_nonneg (sub_nonneg.2 h) r.2
```

Comment:
These are new, right, because they are unbundled and before we didn't have `NNRat` with unbundled ordered algebraic classes? If so, just :+1: and resolve.